src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -332,7 +332,6 @@ def is_ceph_device(lv):
     else:
         return True
 
-
```

Comment:
This is change is unnecessary. Best if reverted.

Reply:
done